cli.md

```diff
@@ -34,9 +34,12 @@ File group(s) used as output.
 
 ## Optional parameters
 
-### `-g, --group-id GROUPID`
+### `-g, --group-id ID`
```

Comment:
Suggestion: -g, --page-id ID
group-id is only understandable if you have the old definition in mind.
Unfortunately -p is already defined.

Reply:
It is clearer true, but we'd have to change a lot of code for it.